packages/application-shell/src/test-utils/test-utils.tsx

```diff
@@ -645,6 +632,7 @@ function renderHook<
     ),
   });
 
+  // @ts-ignore FIXME: prop types of rerender
```

Comment:
It looks like these test utils are also used in downstream applications.  Would it be possible to add the `asyncFocus`/`asyncBlur` methods to this file, similar to what is being done for [react 19 test utils in ui kit](https://github.com/commercetools/ui-kit/blob/f7506a0ba730a3fcd85007842a09bd953eec86cb/test/test-utils.tsx#L59)?

Reply:
Yes, thanks, that's a good point.
I attached those two to `customFireEvent` instead of `fireEvent` because, in app-kit, we re-export everything using:
`export * from '@testing-library/react';`
If I had gone with the following approach:
```ts
import * as rtl from '@testing-library/react';
const { fireEvent: originalFireEvent, ...rtlRest } = rtl;
```
I wouldn't be able to re-export the types from `@testing-library/react`.

That said, please correct me if I'm wrong here or if there’s a better approach.